mu4/notation/view/widgets/editpitch.cpp

```diff
@@ -0,0 +1,70 @@
+//=============================================================================
+//  MusE Score
```

Comment:
This is still the old copyright. I don't know if it will endup being changed. There's many more places though, and its not related to this PR specifically.

Reply:
Thanks, fixed